tests/test_irods.py

```diff
@@ -21,28 +21,32 @@
 import os.path
 from datetime import datetime
 from pathlib import PurePath
-from unittest.mock import patch
 
 import pytest
 from pytest import mark as m
 
 import partisan
+
 from partisan import irods
 from partisan.exception import BatonError, RodsError
 from partisan.irods import (
     AC,
     AVU,
-    Baton,
     Collection,
     DataObject,
     Permission,
```

Comment:
There is an `irods. format_timestamp` in the code. I have the impression you could leave `format_timestamp` here and remove line 30.
```suggestion
    Permission,
    format_timestamp,
```
There is something interesting with the imports in this file. Is it the auto-formatting tool? I see a mix of for example `Collection` and `partisan.irods.Collection`

Reply:
I prefer the qualified `irods.format_timestamp()` because it makes it clear at the call point that it's making a timestamp for iRODS.